src/components/App.js

```diff
@@ -145,6 +147,40 @@ export const App = ({ defaultSource, defaultBabelConfig, defCustomPlugin }) => {
           >
             Use Example (WIP)
           </button>
+          <button
+            onClick={() => {
+              const toSend = {
+                source,
+                plugin_source: enableCustomPlugin ? customPlugin : "",
+                configs: babelConfig.map((config) => JSON.stringify(config)),
+              };
+              const url = "/api/v1/blobs/create";
```

Comment:
Since the server runs on a separate port, I'm thinking we either need to explicitly include this into `url` or we can setup a `proxy` in `package.json`

Reply:
Sounds good! Let's push this to another PR once we get more info on deployment (: